wire-ios-request-strategy/Sources/E2EIdentity/EnrollE2eICertificateUseCase.swift

```diff
@@ -36,7 +36,8 @@ public protocol EnrollE2eICertificateUseCaseInterface {
 /// This class provides an interface to issue an E2EI certificate.
 public final class EnrollE2eICertificateUseCase: EnrollE2eICertificateUseCaseInterface {
```

Comment:
**nitpick(non-blocking):**  
```suggestion
public final class EnrollE2eICertificateUseCase: EnrollE2eICertificateUseCaseProtocol {
```
We use this form on Swift usually. `Interface` usually is a Java Paradigm

Reply:
I agree, but we use `Interface` in some other cases. If you strongly disagree with keeping it, we can implement this refactoring in the next PR and not increase this one. Feel free to open PR with the renaming.